docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,179 @@
+
+The overall heirarchy is described in the the following diagram:
+
+![JdbcPOJOInsertOutputOperator.png](images/jdbcoutput/operatorsClassDiagrams.png)
+
+AbstractTransactionableStoreOutputOperator: A skeleton implementation of an output operator that writes to a transactional store; the tuple type and store type are generic parameters. Defines an input port whose process method invokes the processTuple() abstract method. Exactly-once semantics are not guaranteed and must be provided by subclasses if needed.
+
+AbstractPassThruTransactionableStoreOutputOperator: Simple extension of the above base class wich adds exactly-once semantics by starting a transaction in beginWindow() and committing it in endWindow().
+
+AbstractJdbcTransactionableOutputOperator: (focus of this document) Adds support for JDBC by using an instance of JdbcTransactionalStore as the store. Also adds support for processing tuples in batches and provides an implementation of the proessTuple() abstract method mentioned above.
+
+AbstractJdbcPOJOOutputOperator: Serves as base class for inserting rows in a table using a JDBC store.
+
+**Note**: For enforcing exactly once semantics a table named `dt_meta` must exist in the database. The sample SQL to create the same is as follows
+```
+"CREATE TABLE IF NOT EXISTS dt_meta (dt_app_id VARCHAR(100) NOT NULL, dt_operator_id INT NOT NULL, dt_window BIGINT NOT NULL, UNIQUE(dt_app_id,dt_operator_id,dt_window))".
+```
+**Note**: Additionally this operator assumes that the underlying database/table in which records are to be added supports transactions. If the database/table does not support transactions then a tuple may be inserted in a table more than once in case of auto recovery from a failure (violation of exactly once semantics).
+
+## Operator Information
+1. Operator location: ***malhar-library***
+2. Available since: ***0.9.4***
+3. Java Packages:
+    * Operator: ***[com.datatorrent.lib.db.jdbc.AbstractJdbcTransactionableOutputOperator](https://www.datatorrent.com/docs/apidocs/com/datatorrent/lib/db/jdbc/AbstractJdbcTransactionableOutputOperator.html)***
+
+## How to Use?
+ Concrete subclasses need to choose an appropriate class (such as String or an appropriate concrete java class). Also implement a couple of abstract methods (if not using AbstractJdbcPOJOOutputOperator): `setStatementParameters(PreparedStatement statement, T tuple)` to set the parameter of the insert/update statement (which is a PreparedStatement) with values from the tuple and `getUpdateCommand()` the SQL statement to update a tuple in the database. Note that these methods are also defined in AbstractJdbcPOJOOutputOperator.
```

Comment:
As I mentioned in an earlier comment, the point of the first sentence is not clear: Choose a class for what purpose ? Please explain.

the SQL statement => to return the SQL statement

Note that .... . => Note that subclasses of AbstractJdbcPOJOOutputOperator need not define these methods since they are already defined in that class.